src/openrct2-ui/UiContext.cpp

```diff
@@ -640,6 +640,29 @@ class UiContext final : public IUiContext
         _platformUiContext->ShowMessageBox(_window, message);
     }
 
+    int ShowMessageBox(const std::string& title, const std::string& message, const std::vector<std::string>& options) override
```

Comment:
This should return `int32_t` or `size_t`. What gets returned when you click the X?

Reply:
I don't know yet

Reply:
It returns -1: https://wiki.libsdl.org/SDL_ShowMessageBox

Reply:
In that case, -1 needs to be correctly handled. And the default implementation of UiContext should return -1.